frontend/packages/policy-editor/src/utils/index.ts

```diff
@@ -123,9 +123,10 @@ export const mapPolicyRulesBackendObjectToPolicyRuleCard = (
  */
 export const mapSubjectTitleToSubjectString = (
   subjectOptions: PolicySubject[],
-  subjectTitle: string
+  subjectTitle: string,
 ): string => {
   const subject: PolicySubject = subjectOptions.find((s) => s.subjectTitle === subjectTitle);
+  if (subject.subjectId === '[org]') return `urn:altinn:org:${subject.subjectId}`;
```

Comment:
Burde vi ikke heller oppdatert den faktiske definisjonen av subject-objektet for `[org]`?
Ser at den er satt som 
```
const policySubjectOrg: PolicySubject = {
  subjectDescription: '[org]',
  subjectId: '[org]',
  subjectSource: '[org]',
  subjectTitle: '[org]',
};
```
i query'en der den appendes til listen, tenker det er der vi bør gjøre endringen.

Reply:
Fikser det 😄